### PR TITLE
Doc: Reorganized the work space section, fixing the links and spelling.

### PR DESCRIPTION
**Pull request by parniantaghipour**
The workspace section has been reorganized.

Two of the links were broken due to the change in the file structure.

Change the name of plugins to plugin

### CODE REVIEW

File: docs/features/plugin/start_coding.md
Comment (tjbck) [inline docs/features/plugin/start_coding.md:1]: `getting_started.md`

File: docs/features/plugin/functions/actions.md
Comment (tjbck) [inline docs/features/plugin/functions/actions.md:1]: Too much emojis.

File: docs/features/plugin/functions/filter.md
Comment (tjbck) [inline docs/features/plugin/functions/filter.md:1]: Too much emojis.

File: docs/features/plugin/start_coding.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+---
+sidebar_position: 5
+title: "🔧 Coding Functions"
```
Comment (tjbck) [inline docs/features/plugin/start_coding.md:3]: `Shared Components`, Better suggestions welcome.

File: docs/features/plugin/tools/index.mdx [hunk 2]
```diff
@@ -1,187 +1,68 @@
...
+
+**Tools** make this possible by allowing the LLM to **call external functions** and **retrieve relevant information** or **perform specific actions** during a conversation.
+
+#### 📝 Examples of Tools:
+
+1. **Web Search** 🔍: Get real-time answers from live web searches.
+2. **Image Generation** 🖼️: Create images based on user prompts.
+3. **Voice Synthesis** 🎙️: Integrate with an API like ElevenLabs to generate audio.
+
+## 🔑 How to Enable and Use Tools
```
Comment (tjbck) [inline docs/features/plugin/tools/index.mdx:36]: There are more ways to use tools.

File: docs/features/plugin/tools/index.mdx [hunk 3]
```diff
@@ -1,187 +1,68 @@
...
+## 📥 Installing Tools
+
+You can install Tools from the community in two ways:
+
+1. **Manual Download and Import**
+   - Visit the community site: [https://openwebui.com/tools/](https://openwebui.com/tools/) 🌐
+   - Select a Tool, download it as a JSON export 📄, and then import it in Open WebUI by going to Workspace => Tools and clicking “Import Tools.”
+
+2. **Direct Import via Open WebUI URL**
+   - Go to [https://openwebui.com/tools/](https://openwebui.com/tools/) 🌐, select the Tool you want, and enter the IP of your Open WebUI instance. Click “Import to WebUI” to automatically open your instance and begin the import.
```
Comment (tjbck) [inline docs/features/plugin/tools/index.mdx:55]: This should come first, and is the recommend method.

**Comment by tjbck** [comment]
Images for each features would be useful here.

File: docs/features/plugin/start_coding.md (hunk 1 above)
Comment (parniantaghipour) [reply docs/features/plugin/start_coding.md:3]: I was thinking about some name that will show that it is related to making a new Function. I will change it to this for now.

File: docs/features/plugin/tools/index.mdx (hunk 2 above)
Comment (parniantaghipour) [reply docs/features/plugin/tools/index.mdx:36]: These are examples, how do you suggest changing them?

Comment (tjbck) [reply docs/features/plugin/tools/index.mdx:47]: Mention admins have access to all tools by default, and the following is needed for users with "user" role to access the tools.

File: docs/features/plugin/functions/actions.md
Comment (tjbck) [reply docs/features/plugin/functions/actions.md:1]: Not resolved.

File: docs/features/plugin/functions/filter.md
Comment (tjbck) [reply docs/features/plugin/functions/filter.md:1]: Not resolved.

File: docs/features/plugin/tools/index.mdx (hunk 2 above)
Comment (parniantaghipour) [reply docs/features/plugin/tools/index.mdx:47]: I do not understand that. What is the following? 

and you mean to add it as an example of tools?

File: docs/features/plugin/functions/actions.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ title: "💬 Actions "
 Ever wanted a 🔘 button that lets you quickly do something with the AI’s response? That’s where **Actions** come in. Actions in Open WebUI are mini-interactive elements you can attach to individual chat messages, making interactions smoother and more efficient. ⚡
```
Comment (tjbck) [inline docs/features/plugin/functions/actions.md:8]: Here

File: docs/features/plugin/tools/index.mdx (hunk 2 above)
Comment (tjbck) [reply docs/features/plugin/tools/index.mdx:47]: `🔑 How to Enable and Use Tools` section

**Pull request closed without merging**